users/utils.py

```diff
@@ -1,4 +1,10 @@
-import re
+from datetime import datetime
+import re, jwt
+
+from django.http import JsonResponse
+from users.models import User
+
+from django.conf import settings
```

Comment:
```suggestion
from datetime import datetime
import re, jwt

from django.http import JsonResponse
from users.models import User

from django.conf import settings
```
import 컨벤션에 맞게 수정해 주세요.